features/support/hooks.rb

```diff
@@ -1,3 +1,28 @@
+Before('@ds_testing_table') do
+  ActiveRecord::Base.connection.execute('DROP TABLE IF EXISTS `ds_testings`')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.